src/services/export.service.js

```diff
@@ -13,6 +17,7 @@ const bulkExport = async (request, reply) => {
     const clientEntry = await addPendingBulkExportRequest();
     exportToNDJson(clientEntry, request);
     reply.code(202).header('Content-location', `/bulkstatus/${clientEntry}`).send();
+    updateBulkExportStatus(clientEntry, BULKSTATUS_COMPLETED);
```

Comment:
I don't think this function call will behave as we want it to in practice. This is a function for marking the bulk export job as completed, which should only happen after all the ndjson is exported. Right now, this is kicked off synchronously after the initial response is sent that the export has started.

I think we should move this function call to within the `exportToNDJson` function, and call it with `await` after all the `writeToFile`s have finished.